### ISSUE TITLE
grains.items returns nothing in 2016.3?

### ISSUE THREAD

**Issue by anlutro** [saltstack/salt#31835]
Running off 2016.3 HEAD, all the grains module functions seem to work fine, except the items function. I'm not sure if this is strictly because I'm running salt-ssh or what. `-l trace` shows nothing useful.

```
$ salt-ssh odin grains.items
odin:
    ----------
$ salt-ssh odin grains.get os             
odin:
    Debian
```

**Comment by jfindlay** [comment]
```
centos-7-main ~ master # salt-ssh localhost grains.items
localhost:
    ----------
centos-7-main ~ master # salt-ssh localhost grains.get os
localhost:
    CentOS
```
```yaml
centos-7-main ~ master # salt-ssh --versions
Salt Version:
           Salt: 2016.3.0rc1-1155-g4adc25b

Dependency Versions:
         Jinja2: 2.7.2
       M2Crypto: 0.21.1
           Mako: Not Installed
         PyYAML: 3.10
          PyZMQ: 14.3.1
         Python: 2.7.5 (default, Nov 20 2015, 02:00:19)
           RAET: Not Installed
        Tornado: 4.2.1
            ZMQ: 3.2.5
           cffi: Not Installed
       cherrypy: 3.6.0
       dateutil: Not Installed
          gitdb: 0.6.4
      gitpython: 0.3.6
          ioflo: Not Installed
        libgit2: Not Installed
        libnacl: Not Installed
   msgpack-pure: Not Installed
 msgpack-python: 0.4.6
   mysql-python: 1.2.3
      pycparser: Not Installed
       pycrypto: 2.6.1
         pygit2: Not Installed
   python-gnupg: 0.3.7
          smmap: 0.9.0
        timelib: 0.2.4

System Versions:
           dist: centos 7.2.1511 Core
        machine: x86_64
        release: 3.10.0-327.4.5.el7.x86_64
         system: Linux
        version: CentOS Linux 7.2.1511 Core
```

**Comment by jfindlay** [comment]
I have confirmed that this is specific to salt-ssh on 2016.3.  I have tested `salt-call --local grains.items` on this same branch and `salt-ssh '*' grains.items` on 2015.8.7 and both were successful.

**Comment by cachedout** [comment]
Git bisect leads here: 6838a9534f0dd01291eacfbd6621be43ec0396f4

**Comment by cachedout** [comment]
Fixed by #32111
